vignettes/readr.Rmd

```diff
@@ -142,14 +142,13 @@ mtcars_spec
 cols_condense(mtcars_spec)
 ```
 
-
-By default readr only looks at the first 1000 rows. This keeps file parsing speedy, but can generate incorrect guesses. For example, in `challenge.csv` the column types change in row 1001, so readr guesses the wrong types. One way to resolve the problem is to increase the number of rows:
+When guessing column types, readr selects rows to use for guessing semi-randomly. The default number of rows it selects is 1000. If the column type guess is incorrect, increasing `guess_max` may or may not improve the results.
 
 ```{r}
 x <- spec_csv(readr_example("challenge.csv"), guess_max = 1001)
```

Comment:
Similar to above, if we're trying to make a point about the spec, we should reveal it.